### PR TITLE
Fix the issue GPU skip message(#41378)

### PR DESCRIPTION
**Pull request by chengjinfang1216**
Related #41378 

Fix the issue GPU skip message

### CODE REVIEW

File: torch/testing/_internal/common_distributed.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ def wrapper(*args, **kwargs):
         if not torch.cuda.is_available():
             sys.exit(TEST_SKIPS["no_cuda"].exit_code)
         if torch.cuda.device_count() < int(os.environ["WORLD_SIZE"]):
+            message = "Need at least {} CUDA devices".format(2)
```
Comment (mrshenli) [inline torch/testing/_internal/common_distributed.py:42]: should this be?

```
message = "Need at least {} CUDA devices".format(os.environ["WORLD_SIZE"])
```

Comment (chengjinfang1216) [reply torch/testing/_internal/common_distributed.py:42]: hi @mrshenli ，thanks for point it out， I'll modify it.

**Review by mrshenli — APPROVED** [review]
Thanks for fixing!

**Comment by rohan-varma** [comment]
Sorry for not taking a closer look until now, but I don't think this actually fixes the issue. The dict is modified in the forked subprocess, but the modification does not reflect in the original process which orchestrates the tests.

**Pull request closed without merging**